python/cuxfilter/charts/deckgl/bindings/panel_deck.py

```diff
@@ -0,0 +1,123 @@
+import panel as pn
+import param
+import pydeck as pdk
+
+raw_css = """
+.multi-select {
+color: white;
+z-index: 100;
+background-color: #2c2b2b;
+border-radius: 1px;
+width: 100px !important;
+height: 30px;
+}
+
+.deck-chart {
+    z-index: 10;
+    position: initial !important;
+}
+"""
+
+pn.extension("deckgl", raw_css=[raw_css])
+
+
+class panel_deck(param.Parameterized):
+    x = param.String("x")
+    data = param.DataFrame()
+    colors = param.DataFrame()
+    indices = set()
+    multi_select = param.Boolean(False, doc="multi-select")
+    callback = param.Callable()
+    spec = param.Dict()
+    default_color = param.List([211, 211, 211, 50])
+    sizing_mode = param.String("scale_both")
+    height = param.Integer(400)
+    width = param.Integer(400)
+
+    def get_tooltip_html(self):
+        html_str = ""
+        tooltip_columns = list(
+            set(self.data.columns)
+            - set(["index", "coordinates"] + list(self.colors.columns))
+        )
+        for i in tooltip_columns:
+            html_str += "<b> %s </b>: {%s} <br><br>" % (i, i)
```

Comment:
could use f-string here and avoid old-style positional substitution (tho braces would need escaping so maybe it's a wash)

Reply:
using f-string now